recipe/meta.yaml

```diff
@@ -243,13 +246,10 @@ outputs:
       host:
         - {{ pin_subpackage('libgomp', exact=True) }}
       run:
-        - _libgcc_mutex {{ _libgcc_mutex }}
```

Comment:
Why remove this?

Reply:
It should be in the global pinning files so we do not need it. When I rerendered the recipe however, conda smithy doesn't appear to have picked up that version and put it in the ci_support yaml files.

Reply:
global pinning is used for `host`, not `run`.

Reply:
Well when I rerendered, conda smithy took it out and then the jinja would not work. I don't get why the global pinning does apply here too.

Reply:
The recipe has to be stable under rerendering, right?

Reply:
this is fixed - was the pinning file